configs/terraform/environments/prod/dns-cleaner.tf

```
@@ -0,0 +1,22 @@
+resource "google_service_account" "sa_gke_kyma_integration" {
+  account_id   = "sa_gke_kyma_integration@sap-kyma-prow.iam.gserviceaccount.com"
+  display_name = "sa_gke_kyma_integration"
```

```suggestion
  account_id   = "sa-gke-kyma-integration@sap-kyma-prow.iam.gserviceaccount.com"
  display_name = "sa-gke-kyma-integration@sap-kyma-prow.iam.gserviceaccount.com"
```

You can't use underscore for sa names. See terraform plan error.

```
Error: "account_id" ("sa_gke_kyma_integration@sap-kyma-prow.iam.gserviceaccount.com") doesn't match regexp "^[a-z](?:[-a-z0-9]{4,28}[a-z0-9])$"
```